app/model/Benchmarks.scala

```diff
@@ -24,9 +26,11 @@ object Averages {
   def apply(latestAnalytics: Seq[LatestCampaignAnalytics]): Averages = {
     val noOfAnalytics = latestAnalytics.size.toLong
     Averages(
-      uniques = latestAnalytics.map(_.uniques).sum / noOfAnalytics,
-      pageviews = latestAnalytics.map(_.pageviews).sum / noOfAnalytics,
-      timeSpentOnPage = (latestAnalytics.flatMap(_.weightedAverageDwellTimeForCampaign).sum / noOfAnalytics).to2Dp
+      uniques = Try(latestAnalytics.map(_.uniques).sum / noOfAnalytics).toOption.getOrElse(0),
```

Comment:
If you're using the `Try(operation).toOption.getorElse(0)` pattern a lot, maybe factor it out to a `def sumOrZero`?

Reply:
Good point - will do.